roles/StackStorm.st2/tasks/auth.yml

```diff
@@ -11,12 +11,16 @@
   until: _task is succeeded
   when: ansible_facts.os_family == 'Debian'
 
+- name: Include RedHat OS-specific variables
+  include_vars: st2_{{ ansible_facts.pkg_mgr }}.yml
```

Comment:
Following the logic, EL8 uses `python3-passlib`, while older distros (EL7 and EL6) `python-passlib`.

I think more correct construct would be the following:

```yaml
- name: Include OS specific variables
  include_vars: "{{ item }}"
  with_first_found:
    - "{{ ansible_os_family|lower }}_{{ ansible_distribution_major_version }}.yml"
    - "{{ ansible_os_family|lower }}.yml"
```

`redhat_8.yml`
`redhat.yml`

As it's not really `dnf` vs `yum` thing, but OS version and platform specific logic.